Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -99,6 +100,14 @@ async function run() {
         if(webAppUri) {
             tl.setVariable(webAppUri, publishingProfile.destinationAppUrl);
         }
+		
+		if(renameFilesFlag){
```

Comment:
I have put the logic inside the condition if(utility.canUseWebDeploy(useWebDeploy)) . We now first get the list of app settings, and then toggle the value of the flag using a rest call, only if  the XOR of the appsettings.properties.MSDEPLOY_RENAME_FILES_FLAG and the RenameFiles flag is true.